packages/google_maps_flutter/google_maps_flutter_web/lib/src/google_maps_controller.dart

```diff
@@ -255,30 +255,30 @@ class GoogleMapController {
 
   /// Returns the [LatLngBounds] of the current viewport.
   Future<LatLngBounds> getVisibleRegion() async {
-    return _gmLatLngBoundsTolatLngBounds(await _googleMap.bounds);
+    return _gmLatLngBoundsTolatLngBounds(await _googleMap!.bounds);
   }
 
   /// Returns the [ScreenCoordinate] for a given viewport [LatLng].
   Future<ScreenCoordinate> getScreenCoordinate(LatLng latLng) async {
     final point =
-        _googleMap.projection.fromLatLngToPoint(_latLngToGmLatLng(latLng));
-    return ScreenCoordinate(x: point.x, y: point.y);
+        _googleMap!.projection!.fromLatLngToPoint!(_latLngToGmLatLng(latLng))!;
```

Comment:
That's a lot of `!`...

Reply:
Haha yeah, the underlying API is all nullable `?`, and the previous code did a lot of calls without null-checking in the middle.

There seems to be some wiring that can be done within the map to know when `projection` is ready, but that hasn't been required so far. The rest is still nullable.